layouts/default.vue

```diff
@@ -1,14 +1,26 @@
 <template>
   <div class="layout">
+    <SourcesModal :show="showModal" @close="showModal = false"></SourcesModal>
     <div class="app-wrapper" :class="{ 'show-sidebar': showSidebar }">
       <div class="sidebar">
         <nuxt-link to="/" class="logo"> Open5e </nuxt-link>
+        <div
+          class="bg-red-600 px-4 py-2 cursor-pointer hover:bg-red-400"
+          @click="showModal = true"
+        >
```

Comment:
Yeah, it's the later. It filters all content, site-wide.

My intent was to make it reasonably-prominent since it's a new feature, and it's getting to be pretty important to filter down as we add more sources.

Ironically, search is the _only_ thing not currently covered by it.